### PR TITLE
let rpmbuild know that it should not clean up after itself

### PR DESCRIPTION
**Pull request by xsuchy**
adressing: RHBZ#2105393

### CODE REVIEW

File: mock/py/mockbuild/backend.py [hunk 1]
```diff
@@ -668,7 +668,7 @@ def install_srpm(self, srpm_path):
 
     @traceLog()
     def rebuild_installed_srpm(self, spec_path, timeout):
-        command = ['{command} -bs --target {0} --nodeps {1}'.format(
+        command = ['{command} -bs --noclean --target {0} --nodeps {1}'.format(
```
Comment (praiskup) [inline mock/py/mockbuild/backend.py:671]: This will fail for EL6 targets.

Comment (praiskup) [reply mock/py/mockbuild/backend.py:671]: I would prefer not to use this option as long as "cleanup_on_success" is True.

**Comment by xsuchy** [comment]
Updated.

**Comment by praiskup** [comment]
Looks fine, thank you.

File: mock/py/mockbuild/backend.py (hunk 1 above)
Comment (praiskup) [reply mock/py/mockbuild/backend.py]: A few years later, I think I was wrong: #1809

**Pull request merged**